sqiffy-test/src/test/kotlin/com/dzikoysk/sqiffy/e2e/DslE2ETest.kt

```diff
@@ -188,6 +189,12 @@ internal abstract class DslE2ETest : SqiffyE2ETestSpecification() {
         assertThat(deletedCount).isEqualTo(1)
     }
 
+    @Test
+    fun `should create table with all default values`() {
+        database.generateChangeLog(TestDefaultDefinition::class)
+        TestDefault()
```

Comment:
We need some assertions to verify that this object contains valid values. To avoid repetition, you can extract default values from table definition to const values, so you can reuse them here.